spec/presenters/two_factor_authentication/auth_app_selection_presenter_spec.rb

```diff
@@ -1,12 +1,33 @@
 require 'rails_helper'
 
 describe TwoFactorAuthentication::AuthAppSelectionPresenter do
-  let(:subject) { described_class.new(configuration) }
   let(:configuration) {}
+  let(:user_without_mfa) { create(:user) }
+  let(:user_with_mfa) { create(:user, :with_authentication_app) }
+  let(:presenter_without_mfa) {
+ described_class.new(configuration: configuration, user: user_without_mfa)
+  }
+  let(:presenter_with_mfa) {
+ described_class.new(configuration: configuration, user: user_with_mfa)
+  }
```

Comment:
oops one more of these
```suggestion
  let(:presenter_without_mfa) do
    described_class.new(configuration: configuration, user: user_without_mfa)
  end
  let(:presenter_with_mfa) do
    described_class.new(configuration: configuration, user: user_with_mfa)
  end
```